content/warnings/refs-must-have-owner.md

```diff
@@ -4,45 +4,49 @@ layout: single
 permalink: warnings/refs-must-have-owner.html
 ---
 
-You are probably here because you got one of the following error messages:
+ أنت على الأرجح هُنا بسبب رسائل الخطأ التالية:
 
-*React 16.0.0+*
-> Warning:
+*React 16.0.0 و أعلى*
+> تحذير:
 >
 > Element ref was specified as a string (myRefName) but no owner was set. You may have multiple copies of React loaded. (details: https://fb.me/react-refs-must-have-owner).
+>
+> حُدِد مرجع العُنصر كسلسة نصّية "string" ولكن لم يتم تحديد المالِك "owner". من المُحتمل أن يكون لديك أكثر من نُسخة React مُحمّلة (تفاصيل: https://fb.me/react-refs-must-have-owner).
 
-*earlier versions of React*
-> Warning:
+*النُسخ الأقدم من React*
+> تحذير:
 >
 > addComponentAsRefTo(...): Only a ReactOwner can have refs. You might be adding a ref to a component that was not created inside a component's `render` method, or you have multiple copies of React loaded.
+>
+> addComponentAsRefTo(...): يحظى ReactOwner بالمراجع فقط. من المُحتمل أنك تُضيف مرجع إلى مُكوّن لم يُنشأ داخل تابع `render` أو أنه لديك أكثر من نُسخة React مُحملّة.
```

Comment:
Please comply with the existing translation [glossary](https://github.com/reactjs/ar.reactjs.org/wiki/Glossary) .. 
`ref/refs` stay the same ..

Reply:
I translated about 4 pages where I translated ref/refs ...